README.md

```diff
@@ -1,73 +1,88 @@
-
-General development discussions https://lists.tarantool.org/mailman/listinfo/tarantool-discussions/
-
-Tarantool is an in-memory database and application server.
+It is distributed under [BSD 2-Clause][license] terms.
 
 Key features of the application server:
- * 100% compatible drop-in replacement for Lua 5.1,
-   based on LuaJIT 2.1.
-   Simply use `#!/usr/bin/tarantool` instead of
-   `#!/usr/bin/lua` in your script.
- * full support for Lua modules and a rich set of
-   own modules, including cooperative multitasking,
-   non-blocking I/O, access to external databases, etc
+
+* Heavily optimized Lua interpreter with incredibly fast tracing JIT compiler,
+  based on LuaJIT 2.1.
+* Cooperative multitasking, non-blocking IO.
+* [Persistent queues][queue].
+* [Sharding][vshard].
+* [Cluster and application management framework][cartridge].
+* Access to external databases such as [MySQL][mysql] and [PostgreSQL][pg].
+* A rich set of built-in and standalone [modules][modules].
 
 Key features of the database:
- * ANSI SQL, including views, joins, referential
-   and check constraints
- * MsgPack data format and MsgPack based
-   client-server protocol
- * two data engines: 100% in-memory with
-   optional persistence and an own implementation of LSM-tree, 
-   to use with large data sets
- * multiple index types: HASH, TREE, RTREE, BITSET
- * asynchronous master-master replication
- * authentication and access control
- * the database is just a C extension to the
-   application server and can be turned off
-
-Supported platforms are Linux/x86, FreeBSD/x86 and OpenBSD/x86, Mac OS X.
-
-Tarantool is ideal for data-enriched components of
-scalable Web architecture: queue servers, caches,
-stateful Web applications.
-
-To download and install Tarantool as a binary package for your OS, please visit
-https://tarantool.io/en/download/.
-
-To build Tarantool from source, see detailed instructions in the Tarantool
-documentation at https://tarantool.io/en/doc/2.1/dev_guide/building_from_source/.
-
-Please report bugs at https://github.com/tarantool/tarantool/issues
-We also warmly welcome your feedback in the discussion mailing
-list, tarantool@googlegroups.com.
+
+* MessagePack data format and MessagePack based client-server protocol.
+* Two data engines: 100% in-memory with complete WAL-based persistence and an
+  own implementation of LSM-tree, to use with large data sets.
+* Multiple index types: HASH, TREE, RTREE, BITSET.
+* Document oriented JSON path indexes.
+* Asynchronous master-master replication.
+* Synchronous quorum-based replication.
+* RAFT-based automatic leader election for the single-leader configuration.
+* Authentication and access control.
+* ANSI SQL, including views, joins, referential and check constraints.
+* [Connectors][connectors] for many programming languages.
+* The database is just a C extension to the application server and can be
```

Comment:
```suggestion
* The database is a C extension of the application server and can be
```

Reply:
Your variant sounds better for me, replaced.